docs/CHANGELOG.md

```diff
@@ -6,12 +6,128 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## DevilutionX 1.2.0 (unreleased)
 ### Features
+- Hellfire support
+- Shareware support
+- Fully migrate saves between Diablo and Hellfire
+- Document [ini-file options](https://github.com/diasurgical/devilutionX/wiki)
+- Support file paths longer than 259 characters
+- Allow for up to 99 save games of each type
+- Much lower memory usage (now in line with the original)
+- Performance improvements
+- Gold picked goes to stack even when inventory is open
```

Comment:
Actually didn't know we implemented this 😆

Reply:
Well technically it's brought over from Hellfire :)